Documentation/Concepts/Backend/PageModule/Index.rst

```diff
@@ -97,3 +97,24 @@ need some experience in PHP programming and dealing with databases etc.
 
     Other content management systems like wordpress use the word "plugin" to
     describe what we call an :ref:`extension <concepts-extensions>` in TYPO3.
+
+..  _preview-page:
+
+How to preview a page
+=====================
+
+The preview function in TYPO3 allows you to check pages before publishing them.
+
+#.  Preview from the Page Module
+
+    *   You can preview the page by clicking on the desired page in the page
```

Comment:
I miss a note about the context menu. It will not open by just clicking on the page in page tree. You have either to click or right-click on the page icon or IMO since TYPO3 12 or 13 you can reach the context menu also by right click on page label in page tree.